fmriprep/workflows/bold/outputs.py

```diff
@@ -338,6 +339,23 @@ def init_func_derivatives_wf(
                                         (('cifti_metadata', _read_json), 'meta_dict')])
         ])
 
+    if "compcor" in config.execution.debug:
+        ds_acompcor_masks = pe.Node(
+            DerivativesDataSink(
+                base_directory=output_dir, suffix="mask", desc=["CSF", "WM", "acompcor"],
+                compress=True),
+            name="ds_acompcor_masks", run_without_submitting=True)
```

Comment:
If my suggestion above is taken, I think these two data sinks could be reduced to just one.

Reply:
We'd also need a Merge node, so it's very little difference in verbosity, and I think explicit will be clearer.